core/templates/dev/head/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -221,8 +228,11 @@ angular.module('oppia').directive('contributionsAndReview', [
                       {
                         action: SuggestionModalService.ACTION_ACCEPT_SUGGESTION,
                         commitMessage: $scope.commitMessage,
-                        reviewMessage: $scope.reviewMessage
+                        reviewMessage: $scope.reviewMessage,
+                        skillDifficulty: $scope.difficulty
                       });
+                    AlertsService.addSuccessMessage(
+                      'Accepted question suggestion.');
```

Comment:
Move this to the success handler of the HTTP call where you make https request. (Maybe rename the removeContributionToReview and all the required stuff there or move the Alert to the `ContributionAndReviewService`, I would suggest the 1st one as we will be changing the ContributionAndReviewService to ContributionAndReviewBackendApiService in future.)